test/.dapr/components/pubsub.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
```

Comment:
Just wondering, as it is now here - of course because of testing purposes - but shouldn't the whole `.dapr` folder be part of the runtime package?

Reply:
Good idea, but having this should be a separate task

Reply:
If not before this, it will be moved by the AppManifest change. But might as well be moved before that is released.